File: CS/MyWizardControl/MyNextButton/MyBackButtonViewInfo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraWizard;
using DevExpress.XtraEditors.Drawing;
using DevExpress.Skins;

namespace WindowsApplication1
{
    public class MyBackButtonViewInfo : BackButtonViewInfo
    {
        public MyBackButtonViewInfo(BackButton owner)
            : base(owner)
        {

        }

        protected override EditorButtonPainter GetButtonPainter()
        {
            return new MyButtonPainter(this.LookAndFeel);
        }
    }
}

File: VB/MyWizardControl/MyNextButton/MyBackButtonViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraWizard
Imports DevExpress.XtraEditors.Drawing
Imports DevExpress.Skins

Namespace WindowsApplication1
	Public Class MyBackButtonViewInfo
		Inherits BackButtonViewInfo
		Public Sub New(ByVal owner As BackButton)
			MyBase.New(owner)

		End Sub

		Protected Overrides Function GetButtonPainter() As EditorButtonPainter
			Return New MyButtonPainter(Me.LookAndFeel)
		End Function
	End Class
End Namespace
